packages/ketcher-core/src/application/render/restruct/retext.ts

```diff
@@ -50,17 +51,19 @@ class ReText extends ReObject {
     return true
   }
 
-  getReferencePoints(): Array<Vec2> {
+  getReferencePoints(): Array<Vec2> | null {
+    if (!this.paths.length) return null
```

Comment:
let's return an empty array here to avoid potential runtime error.

Reply:
Fixed